DOCSP-36266 Add Gen AI FAQs (#159)
mongodb/docs-landing: source/generative-ai-faq.txt

:template: landing
:nav-show-all-products: true
:title: Generative AI FAQs

==================
Generative AI FAQs
==================

FAQ For MongoDB Generative AI Features
--------------------------------------

What GenAI Features Does MongoDB Currently Offer?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

MongoDB offers various generative AI (GenAI) capabilities in its 
products and services, including within Atlas and in its software 
products. 
For example, in MongoDB Compass and Charts, Atlas users can
leverage natural language to generate queries or visualizations. And 
with MongoDB Relational Migrator, GenAI can be leveraged for query 
conversions.

We also offer a GenAI chatbot within our MongoDB Documentation. 

What Services Do These GenAI Features Rely On?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> MongoDB's current GenAI features are powered by Azure OpenAI. When you 

Reviewer: ```suggestion
MongoDB's current GenAI features are powered by [Azure OpenAI](https://azure.microsoft.com/en-us/products/ai-services/openai-service). When you 
```

use these features, your data is processed in Microsoft Azure using 
the Azure OpenAI service. 

Data processed in Azure OpenAI is not used 

> for any model training. MongoDB has opted out of asynchronous Abuse 
> Monitoring so that no prompts or responses are stored with Azure OpenAI. 

Reviewer: ```suggestion
for any model training. MongoDB has opted out of [asynchronous Abuse 
Monitoring](https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/abuse-monitoring) so that no prompts or responses are stored with Azure OpenAI. 
```

> For more detail, refer to the Azure OpenAI data management policy.

Reviewer: We can link if we reword it to:
```suggestion
For more detail, refer to [Azure OpenAI's data management practices](https://learn.microsoft.com/en-us/legal/cognitive-services/openai/data-privacy).
```

information to produce the best possible answer. To learn more 
about what each tool sends, please consult our documentation:

- `Compass <https://dochub.mongodb.org/core/compass-ai-data-usage>`__
- `Atlas Charts <https://dochub.mongodb.org/core/charts-ai-data-usage>`__
- `Relational Migrator <https://dochub.mongodb.org/core/migrator-ai-data-usage>`__

Are Outputs from my Use of MongoDB GenAI Features Used to Train Models?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

No.

Does MongoDB Store my Inputs or their Corresponding Outputs to GenAI Features?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

MongoDB will store inputs and outputs from the GenAI service for a 
limited time period (up to one year) for troubleshooting and analytics. 
For any GenAI capabilities offered in our on-premise software 
(Relational Migrator and Compass) you may opt out, in which case
MongoDB will not store your inputs or outputs.

What Agreements Govern My Use of MongoDB's GenAI Features?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Your use of these features is subject to your applicable MongoDB legal 
agreement, our

> `Acceptable Use Policy <https://www.mongodb.com/legal/terms-of-use>`_ and 

Reviewer: Incorrect link:
```suggestion
`Acceptable Use Policy <https://www.mongodb.com/legal/acceptable-use-policy>`_ and 
```

:template: landing
:nav-show-all-products: true
:title: Generative AI FAQs

==================
Generative AI FAQs
==================

FAQ For MongoDB Generative AI Features
--------------------------------------

What GenAI Features Does MongoDB Currently Offer?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

MongoDB offers various generative AI (GenAI) capabilities in its products 

> and services, including within Atlas and in its software products. 

Reviewer 2: that makes sense! Since "its software products" is covered earlier in the sentence ("in its products and services"), I would suggest taking out software products for minimalism. 

```suggestion
and services, including within Atlas.
```

> and services, including within Atlas and in its software products. 

Reviewer 2: Or alternatively, 
```suggestion
and services.
```

For example, in MongoDB Compass, and Charts, Atlas users can 
leverage natural language to generate queries or visualizations. 
And with MongoDB Relational Migrator, GenAI can be leveraged for 
query conversions.

We also offer a GenAI chatbot within our MongoDB Documentation. 

What Services Do These GenAI Features Rely On?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

MongoDB's current GenAI features are powered by `Azure OpenAI 
<https://azure.microsoft.com/en-us/products/ai-services/openai-service>`__. 
When you use these features, your data is processed in Microsoft 
Azure using the Azure OpenAI service. 

Data processed in Azure OpenAI is not used for any model training. 
MongoDB has opted out of `asynchronous Abuse Monitoring 
<https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/abuse-monitoring>`__ 
so that no prompts or responses are stored with Azure OpenAI. For more detail, 
refer to the `Azure OpenAI data management policy 
<https://learn.microsoft.com/en-us/legal/cognitive-services/openai/data-privacy>`__.

> What Data Is Being Sent To The Model?
> ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Reviewer 2: [suggestion] I think we could shorten this for minimalism

```suggestion
What Data Is Sent To The Model?
~~~~~~~~~~~~~~~~~~~~~~~~
```

Each MongoDB tool backed by GenAI sends the minimal required 
information to produce the best possible answer. To learn more 
about what each tool sends, please consult our Documentation:

- `Compass <https://dochub.mongodb.org/core/compass-ai-data-usage>`__
- `Atlas Charts <https://dochub.mongodb.org/core/charts-ai-data-usage>`__
- `Relational Migrator <https://dochub.mongodb.org/core/migrator-ai-data-usage>`__

Are Outputs from my Use of MongoDB GenAI Features Used to Train Models?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

No.

Does MongoDB Store my Inputs to GenAI Features or the Corresponding Outputs?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

MongoDB stores inputs and outputs from the GenAI service for 
a limited time period (up to one year) for troubleshooting and 
analytics. For any GenAI capabilities offered in our on-premise 
software (Relational Migrator) you may opt out, in which case 

> MongoDB does not store inputs or outputs from the model.

Reviewer 2: [question] (for my own limited knowledge of GenAI tools) Can you clarify the difference between inputs/outouts of the GenAI service versus inputs/outputs of the model?

:template: landing
:nav-show-all-products: true
:title: Generative AI FAQs

==================
Generative AI FAQs
==================

FAQ For MongoDB Generative AI Features
--------------------------------------

What GenAI Features Does MongoDB Currently Offer?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

MongoDB offers various Generative AI (GenAI) capabilities in its products 
and services, including within Atlas and in its software products. 

> For example in MongoDB Compass users can 

Reviewer 2: [nit]

```suggestion
For example, in MongoDB Compass users can 
```

leverage natural language to generate MongoDB queries.

> In Charts, Atlas users have the capability to construct 
> visualizations powered by GenAI. Additionally, 

Reviewer 2: [suggestion] we could probably make this more active as well

```suggestion
In Charts, Atlas users have the capability to construct 
GenAI powered visualizations. Additionally, 
```

so that no prompts or responses are stored within Azure OpenAI. For more details, 
refer to the `Azure OpenAI data management policy 
<https://learn.microsoft.com/en-us/legal/cognitive-services/openai/data-privacy>`__.

What Data Is Sent To The Model?
~~~~~~~~~~~~~~~~~~~~~~~~

Each MongoDB tool backed by GenAI sends the minimal required 
information to produce the best possible answer. To learn more 
about what each tool sends, please consult our Documentation:

- `Compass <https://dochub.mongodb.org/core/compass-ai-data-usage>`__
- `Atlas Charts <https://dochub.mongodb.org/core/charts-ai-data-usage>`__
- `Relational Migrator <https://dochub.mongodb.org/core/migrator-ai-data-usage>`__

Are Outputs from my Use of MongoDB GenAI Features Used to Train Models?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

No.

Does MongoDB Store my Inputs or their Corresponding Outputs to GenAI Features?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

MongoDB stores inputs and outputs from the GenAI service for 
up to one year for troubleshooting and 
analytics. For any GenAI capabilities offered in our on-premise 

> software (Relational Migrator and Compass) you may opt out of input 

Reviewer 2: [nit]

```suggestion
software (Relational Migrator and Compass), you may opt out of input 
```
